islandora_pathauto.module

```diff
@@ -187,6 +190,13 @@ function islandora_pathauto_pathauto_bulkupdate(&$context) {
       $filter = "sameTerm(?model, <info:fedora/$filter>)";
     }
     $sandbox['filters']['model'] = format_string('FILTER(!models)', array('!models' => implode(' || ', $model_filters)));
+    if (variable_get('islandora_namespace_restriction_enforced', FALSE)) {
```

Comment:
@nhart could you update this part of the code so we can get this into the release?I really would prefer that implementation I referenced. I think we will need to make a project wide check on namespace restrictions probably in the future, and makes sense to have if not a common function, at least the same implementation everywhere to  make searching and debugging easier. thanks a lot!!